requirements/app/base.txt

```diff
@@ -1,5 +1,5 @@
 lightning-cloud>=0.5.10
-packaging
+typing-extensions>=4.0.0, <4.3.1
```

Comment:
Seems like this is causing some problems. Any idea @otaj?
https://github.com/Lightning-AI/lightning/actions/runs/3241245486/jobs/5312932676#step:12:38

Reply:
What? How?

Reply:
I mean, I see that we are failing on this step, but wtf?

Reply:
Once again, we *NEED* to have tests for the magic in `assistant.py` and `setup_tools.py`. This error is caused by this: https://github.com/Lightning-AI/lightning/actions/runs/3241541479/jobs/5313622020#step:6:12